### PR TITLE
Tests permission fix

### PR DESCRIPTION
**Pull request by dmankins**
In copying files from our internal git repo, I neglected to set permissions correctly on the shell-scripts (and also missed a file!).

While I was at it, I hope I clarified the README a little, and added a related comment to the Makefile.

.... and I also removed a stone from the shoe of a tester (make sure vsql is in the path before spending time starting up the container, only to fail to run the tests).

### CODE REVIEW

File: one-node-ce/README.md [hunk 1]
```diff
@@ -91,11 +95,13 @@ After `Dockerfile_<distro>` installs the RPM or DEB file, it runs `tools/cleanup
 
 # Testing with ./run_tests.sh
 
+Once you have built your container, you can test it using the `./run_tests.sh` script (or by running `make test`).
+
 The `./run_tests.sh` script verifies that the container can execute Vertica and some of the additional libraries. This script requires a [local copy of the vsql client](#getting-a-local-copy-of-vsql). You can run the script directly, or you can run it as `make test`.
```
Comment (spilchen) [inline one-node-ce/README.md:100]: The last sentence in line 100 is made redundant by the addition at line 98.  Maybe consider removing it?

File: one-node-ce/README.md [hunk 2]
```diff
@@ -91,11 +95,13 @@ After `Dockerfile_<distro>` installs the RPM or DEB file, it runs `tools/cleanup
...
 
 Before you test your container, you must stop any existing Vertica server (container or otherwise) on your test system because the `./run_tests.sh` script uses the normal Vertica port number.
 
-The test creates a new container with a unique tab, and volume. Because the test sets up the optional libraries and creates the VMart database, creating a new container can take as long as three minutes.
+The test uses your image to create a new container with a unique tab, and volume. Because the test sets up the optional libraries and creates the VMart database, creating a new container can take as long as three minutes.
```
Comment (spilchen) [inline one-node-ce/README.md:104]: "new container with a unique tab," what is a tab?

**Review by spilchen — COMMENTED** [review]
Looks good.  I just had minor comments on the README.

File: one-node-ce/README.md (hunk 1 above)
Comment (dmankins) [reply one-node-ce/README.md:100]: Good point.  Thanks

File: one-node-ce/README.md (hunk 2 above)
Comment (dmankins) [reply one-node-ce/README.md:104]: should be "tag"

**Pull request merged**